scripts/README.md

```diff
@@ -1,13 +1,33 @@
-#Loading data for development
+# Loading data for development
```

Comment:
I updated the data loading docs to provide a little more background on the requirements. Apologies for the ugly diff.